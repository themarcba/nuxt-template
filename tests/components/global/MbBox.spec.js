import {} from 'jest';
// Doc: https://vue-test-utils.vuejs.org/api/
import { createLocalVue, config, shallowMount } from '@vue/test-utils';
import MbBox from '@/components/global/MbBox.vue';
import NuxtStubs from '@/tests/nuxt-stubs';

const localVue = createLocalVue();

// Doc: https://vue-test-utils.vuejs.org/api/#config
config.showDeprecationWarnings = true;

describe('MbBox.vue', () => {
  const wrapper = shallowMount(MbBox, {
    localVue,
    stubs: NuxtStubs
  });

  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy();
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
