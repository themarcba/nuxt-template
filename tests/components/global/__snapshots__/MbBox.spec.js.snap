// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MbBox.vue matches the snapshot 1`] = `<div class="bg-elevatedSurface text-primary rounded-md shadow-md p-4"></div>`;
